hack/verify-kustomize.sh

```diff
@@ -15,44 +15,46 @@
 # limitations under the License.
 
 _GOPATH_BIN="$(go env GOPATH)/bin"
-_MIN_KUSTOMIZE_VERSION="v3.3.0"
+_TMP_BINDIR="/tmp/antrea"
+# Must be an exact match, as the generated YAMLs may not be consistent across
+# versions
+_KUSTOMIZE_VERSION="v3.8.2"
```

Comment:
any reason for 3.8.2 and not 3.8.1 as min?

Reply:
3.8.2 is the latest patch release version and includes bug fixes. BTW I updated the PR to always download kustomize regardless of whether it is already installed in the system, since the version pre-installed in the Github worker VMs (which in practice turns out to be v3.8.2, and not v3.8.1, despite what the documentation says...) uses an Ubuntu snap, which creates other issues.